src/rust/plugin-registry/src/server/deploy_plugin.rs

```diff
@@ -167,6 +167,30 @@ pub async fn deploy_plugin(
     Ok(())
 }
 
+#[tracing::instrument(skip(client, db_client, plugin), err)]
+pub async fn teardown_plugin(
```

Comment:
Unspecified behavior: what do we return if the plugin is already torn down?

Reply:
fixed